app/src/components/account/UserMenu.vue

```diff
@@ -40,25 +40,16 @@
           @click="selectAddress(address)"
         >
           <div class="address-item">
-            <div
-              v-if="getAddressNetwork(address).testnet"
-              class="testnet-icon-container"
-            >
-              <img class="testnet-icon" src="/img/icons/test-tube-32.png" />
-            </div>
```

Comment:
I think it is a good idea to have something visual to tell the testnets apart. Maybe not this particular test tube, but something similar. I think it is a good idea

Reply:
ping @jbibla

Reply:
lol i didn't realize it was a test tube. that's cute. we are writing the network title - so it will say "Testnet" if it's a testnet. 

> It is always better to work with icons and visual stuff than with text. Visual gets easier into people's brain

according to who / what / why

Reply:
Mmm, I think it is better to have something visual here because otherwise you need to be reading the titles. But OK :shrug: 

Let's wait for some user to ask for this, like it happened in the end with the 'nite' mode :smirk: